src/services/authentication/index.js

```diff
@@ -8,17 +8,24 @@ const { hooks } = require('./hooks');
 const { authConfig } = require('./configuration');
 
 class SCAuthenticationService extends AuthenticationService {
+	async getUserData(userId, accountId) {
+		const user = await this.app.service('usersModel').get(userId);
+		const account = await this.app.service('accountModel').get(accountId);
```

Comment:
this call to accounts is added ,can we avoid this call when we have this data available in authResult.account already? this could be done, when we take full user and acocunt instead of ids as params and fetch both from outside.

has the systemId been removed? i think it should stay.